Gruntfile.coffee

```diff
@@ -154,6 +154,13 @@ module.exports = (grunt) ->
           from: /The current version is .*\..*\..*./
           to: "The current version is <%=pkg.version%>."
         }]
+      sourceMap:
+        src: ['dist/js/ember-widgets.min.js', 'dist/js/ember-widgets.js'],
+        overwrite: true,
+        replacements: [{
+          from: '//@ sourceURL',
```

Comment:
What actually introduces the source map url like this? Seems like we should fix that specifically instead of just remapping stuff.